framework/components/ecl-forms/ecl-forms-legends/README.md

```diff
@@ -1 +1,4 @@
-# Legends
```

Comment:
In other components we kept the global title, so I assume we should keep it here too (and set the other title as second level title)